src/test/java/net/snowflake/client/jdbc/SnowflakeDriverLatestIT.java

```diff
@@ -852,12 +852,13 @@ private void testGeoMetadataSingle(
   }
   /**
    * Tests that upload and download small file to gcs stage. Require SNOW-206907 to be merged (still
-   * pass when not merge, but will use presigned url instead of GoogleCredential)
+   * pass when not merge, but will use presigned url instead of GoogleCredential) SNOW-206907 PR is
+   * merged, so enabling the test.
    *
    * @throws Throwable
    */
   @Test
-  @Ignore
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
```

Comment:
Just to double check that this is what we want. `run it on Jenkins but not on github`

Reply:
Yes. All GCS tests are not running in Jenkins. Only AWS tests are running in Jenkins.